Giveaways.md

```diff
@@ -1,15 +1,36 @@
 # Giveaways
-From time to time DevStaff has the opportunity to offer some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
+From time to time DevStaff has the opportunity to offer some giveaways during 
+the meetups. The prizes are provided by [sponsors]
+(https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) 
+in return of advertising.
```

Comment:
in exchange for promotion.

Reply:
Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/494/commits/d5c520d78b77eee7c7e05dc4df9ee9a3792c41f3